docs/contributing.md

```diff
@@ -274,10 +274,9 @@ In order to issue a new release, follow the following steps:
 
         make compress-assets
 
--   Build the translations:
+-   Extract the translations:
```

Comment:
Reading this, I'm wondering if we shouldn't rename the Makefile target to `extract-translations` instead?